lib/classes/CLI.js

```diff
@@ -16,13 +16,18 @@ class CLI {
     this.serverless = serverless;
     this.inputArray = inputArray || null;
     this.loadedPlugins = [];
+    this.loadedExternalPlugins = new Set();
```

Comment:
Let's not introduce this collection (we can simply access the external plugins from plugin manager via `this.serverless.pluginManager.externalPlugins`)

I understand that you've tried to maintain the existing convention, however some of the conventions as found in the code-base are old, unnecessarily verbose (imply maintenance cost, which could be avoided), and we try to not follow them with new code.